test_security/CMakeLists.txt

```diff
@@ -269,9 +269,10 @@ if(BUILD_TESTING)
     endif()
 
     # TODO(mikaelarguedas) only Connext and FastRTPS support DDS-Security for now
+    # TODO(jacobperron) Disable Connext on Windows until we fix issue with CI
     if(
-      rmw_implementation STREQUAL "rmw_connext_cpp" OR
-      rmw_implementation STREQUAL "rmw_connext_dynamic_cpp" OR
+      rmw_implementation STREQUAL "rmw_connext_cpp" AND NOT WIN32 OR
+      rmw_implementation STREQUAL "rmw_connext_dynamic_cpp" AND NOT WIN32 OR
```

Comment:
can you add parentheses? --it's not clear to me how `AND/OR` precedence works--

Reply:
cdd83f8